TCAT/Controllers/InformationViewController.swift

```diff
@@ -42,138 +43,137 @@ class InformationViewController: UIViewController, UITableViewDataSource, UITabl
 
     ]
 
+    private let descriptionLabelSize = CGSize(width: 167, height: 34)
+    private let descriptionLabelTopOffset: CGFloat = 12
+    private let headerWidth = UIScreen.main.bounds.width
+    /** Represents the total height of the headerView given the width of the phone screen. Most of the variables
+     used are static except for the tcat image height, which is dependent on the width of the user's screen.
+     The extra 37 is used for bottom inset padding for the headerView, but I felt it would be unnecessary to
+     create a variable for it */
```

Comment:
In terms of maintaining this codebase for the future, the last sentence (beginning with `The extra 37...`) should be deleted, I don't think anyone will understand when you write `I felt it would be unnecessary...`